site/oauth2-examples-proxy.md

```diff
@@ -0,0 +1,75 @@
+# Use OAuth2 Proxy and Keycloak as OAuth 2.0 server
+
+Let's test the following flow:
+
+* Access management ui via a browser thru OAuth2-Proxy
```

Comment:
ok update the comment to read "Access the RabbitMQ Management UI using a browser through OAuth2-Proxy"